types/screen.d.ts

```diff
@@ -13,7 +13,7 @@ export type Screen<Q extends Queries = typeof queries> = BoundFunctions<Q> & {
     options?: OptionsReceived,
   ) => void
   /**
-   * Convenience function for `Testing Playground` which logs URL that
+   * Convenience function for `Testing Playground` which logs & return URL that
    * can be opened in a browser
    */
   logTestingPlaygroundURL: (element?: Element | HTMLDocument) => void
```

Comment:
Because it returns the URL.

```suggestion
  logTestingPlaygroundURL: (element?: Element | HTMLDocument) => string
```